_posts/2021-12-21-wt-open-research-fund.md

```diff
@@ -0,0 +1,47 @@
+- bebatut
+- emmyft
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1533930086187-0fc58e5a92e2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987&q=80
+photos:
+  name: Photo by Elena Koycheva on Unsplash
+  url: https://unsplash.com/@lenneek
+---
+
+# Open Life Science is awarded Wellcome Trust Open Research Fund 2021
+
+We are delighted to share that Wellcome Trust has awarded the Open Research Fund to Open Life Science (OLS) to enable research and sustainability of the program for the next two years. Under the title **Systematic evaluation of community development through open science training and incentivising contextual mentorship in health research**, OLS was granted **GBP 99,000** to create a funded position for a programme coordinator and researcher, as well as pay for the overhead cost for the next four cohorts. 
+
+The full proposal (personal information redacted) can be found on Zenodo: [DOI: 10.5281/zenodo.5267934](https://zenodo.org/record/5267934). Below we share some highlights from the proposal and OLS plans for the next two years.
+
+## Goals and Outcomes
+
+Our main goal with this proposal is to conduct a research-based assessment, decolonisation and dissemination of our resources for their adoption and reuse across different open science communities. A successfully completed Open Research Fund activity in OLS will enable:
+
+1. research-based evaluation and enhancement of the OLS curriculum, training materials, mentoring formats and value-based approach to local community development,
+1. incentivised integration of contextual (sociotechnical and cultural) knowledge of mentors and project leads in the open science project they develop in the program,
+1. enhanced representation and equitable participation of health researchers and community builders from the Global South in a shared vision for the global open science through collaboration,
+1. assessment of the long-term impact of OLS on the careers of the OLS participants and adoption of contextualised open science practices in research work in their regional contexts across low- and middle-income, as well as high-income countries, and
+1. publication of a series of reports capturing different outputs, building an open source evidence base and inviting support for the sustainability of OLS over the next 5 years.
+
+## Funded Position: Programme Coordinator and Researcher
+
+To maximise the impact of our project, we will recruit a staff member to join the OLS team as a Programme Coordinator and Researcher. They will also provide administrative support such as by helping ensure transparent communication with all the stakeholders, participating in outreach efforts, processing feedback from the cohorts and developing reports to share openly. Working with the OLS leadership team, they will also co-lead the impact studies and evaluation of OLS training and resources. A job announcement will be made in February 2022. 
```

Comment:
```suggestion
To maximise the impact of our project, we will recruit a staff member to join the OLS team as a Programme Coordinator and Researcher. They will also provide administrative support such as by helping ensure transparent communication with all the stakeholders, participating in outreach efforts, processing feedback from the cohorts and developing reports to share openly. Working with the OLS leadership team, they will also co-lead the impact studies and evaluation of OLS training and resources. A job vacancy will be advertised in February 2022. 
```

to make it clear we're not announcing who was selected :)

Reply:
good point